spec/models/reservations_validations_spec.rb

```diff
@@ -6,6 +6,8 @@
 
   let(:now) { Time.zone.now }
 
+  it { is_expected.to validate_presence_of(:reserve_start_at) }
```

Comment:
I thought about doing that, but the factory setup needed to make that work is pretty fussy.